src/olympia/bandwagon/tests/test_views.py

```diff
@@ -1426,6 +1410,208 @@ def test_404(self):
         assert response.status_code == 404
 
 
+class CollectionViewSetDataMixin(object):
+    client_class = APITestClient
+    data = {
+        'name': {'fr': u'lé $túff', 'en-US': u'$tuff'},
+        'description': {'fr': u'Un dis une dát', 'en-US': u'dis n dat'},
+        'slug': u'stuff',
+        'public': True,
+        'default_locale': 'fr',
+    }
+
+    def setUp(self):
+        self.url = self.get_url(self.user)
+        super(CollectionViewSetDataMixin, self).setUp()
+
+    def send(self, url=None, data=None):
+        pass
+
+    def get_url(self, user):
+        pass
```

Comment:
They're implemented in the tests Classes that inherit from the mixin.  I'll have them throw a method not implemented error.